src/main/java/org/mamute/model/Tag.java

```diff
@@ -48,7 +54,8 @@ public Tag() {
 	
 	public Tag(String name, String description, User author) {
 		this.name = name.toLowerCase();
-		this.description = description;
+        this.sluggedName = toSlug(name);
+        this.description = description;
```

Comment:
I think we can create a "Contributing" section in our readme or a question at meta.mamute.org for that...